### PR TITLE
fix idle resolutions

### PR DESCRIPTION
**Pull request by AjayTripathy**

### CODE REVIEW

File: costmodel/router.go [hunk 1]
```diff
@@ -440,16 +441,11 @@ func (a *Accesses) AggregateCostModel(w http.ResponseWriter, r *http.Request, ps
 		return
 	}
 
-	// if custom pricing has changed, then clear the cache and recompute data
-	if a.CustomPricingHasChanged() {
```
Comment (dwbrown2) [inline costmodel/router.go]: did you mean to remove this?

File: costmodel/router.go [hunk 2]
```diff
@@ -533,35 +532,40 @@ func (a *Accesses) AggregateCostModel(w http.ResponseWriter, r *http.Request, ps
...
 	}
 
-	c, err := a.Cloud.GetConfig()
+	c, err := A.Cloud.GetConfig()
 	if err != nil {
 		w.Write(WrapData(nil, err))
 		return
 	}
-	discount, err := parsePercentString(c.Discount)
+	discount, err := strconv.ParseFloat(c.Discount[:len(c.Discount)-1], 64)
```
Comment (dwbrown2) [inline costmodel/router.go]: niko just added this. why are we removing?

File: costmodel/router.go [hunk 3]
```diff
@@ -441,6 +441,10 @@ func (a *Accesses) AggregateCostModel(w http.ResponseWriter, r *http.Request, ps
 		return
 	}
 
+	if a.CustomPricingHasChanged() {
```
Comment (dwbrown2) [inline costmodel/router.go]: why still lower case?

File: costmodel/router.go (hunk 1 above)
Comment (AjayTripathy) [reply costmodel/router.go]: Fixed.

File: costmodel/router.go (hunk 2 above)
Comment (AjayTripathy) [reply costmodel/router.go]: Fixed, uses ParsePercentString now.

File: costmodel/router.go (hunk 3 above)
Comment (AjayTripathy) [reply costmodel/router.go]: Fixed.

**Pull request merged**